### DISCUSSION TITLE
How to apply datacamp 3 months free trial?

### DISCUSSION THREAD

**Discussion by NyeinHsuThar** [community/community#29527] [Other Feature Feedback, Questions, & Ideas] [answered]
I am a final year student at a computer university in my country. I am trying to get 3 months free trial on datacamp but my school ID is written in our local language and didn't match their requirements. 
Please How to get datacamp 3 months trial?

**Comment by queenofcorgis** [comment]
Hi @NyeinHsuThar,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on https://www.datacamp.com/. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

**Comment by Shaownak** [comment, marked as answer]
Try using the .edu mail address provided by your university.

**Comment by Liza-Shtonda** [comment]
Hi everyone, I wanted to get free trial on datacamp but I have problem with confirmation my GitHub. After log in there is a message: {"message":"no Route matched with those values"}. Hale can I get free trial with github?

**Reply by Shaownak** [reply]
For getting 3 months free trial with GitHub, you have to avail the GitHub student starter pack.

**Reply by Zoltus** [reply]
Datacamp support told me "Please be advised that we are re-evaluating the way the GitHub trial integrates with DataCamp. For that reason when trying to activate the trial plan you might be experiencing issues. We will review your account status and evaluate your eligibility manually." I had to send them screenshots that im github student from my profile

**Reply by ndiimanae** [reply]
I'm experiencing the same issue

**Comment by Liza-Shtonda** [comment]
And how long had  you waited for answer from support team?
сб, 20 серп. 2022 р. о 22:15 Noa T ***@***.***> пише:

**Reply by Zoltus** [reply]
> And how long had you waited for answer from support team? сб, 20 серп. 2022 р. о 22:15 Noa T ***@***.***> пише:
> […](#)
> Datacamp support told me "Please be advised that we are re-evaluating the way the GitHub trial integrates with DataCamp. For that reason when trying to activate the trial plan you might be experiencing issues. We will review your account status and evaluate your eligibility manually." — Reply to this email directly, view it on GitHub <https://github.com/orgs/community/discussions/29527#discussioncomment-3438391>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ASXWABMKVDWR33ZBIUUKKXTV2EVFZANCNFSM56SHZW5Q> . You are receiving this because you commented.Message ID: ***@***.***>

1day

**Comment by seifhassine** [comment]
My university blocks most verification emails for some reason, so I can't use my university email on any website (including github). I'm using my gmail account, and  I get the {"message":"no Route matched with those values"} message.

**Reply by Zoltus** [reply]
> My university blocks most verification emails for some reason, so I can't use my university email on any website (including github). I'm using my gmail account, and I get the {"message":"no Route matched with those values"} message.

If you have github student status you will get datacamp student aswell. As long as you can give datacamp proof that u have github student  u will get it.

**Reply by seifhassine** [reply]
A hotfix has been pushed and now I could successfully get my subscription. Thanks.
